bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,708 @@
+        <item-type>Number</item-type>
+        <label>Brightness of LED during the day</label>
+    </channel-type>
+    
+    <channel-type id="nightmode_BrightnessNight" advanced="false">
+        <item-type>Number</item-type>
+        <label>Brightness of LED at night</label>
+    </channel-type>
+    
+    <channel-type id="nightmode_FanNightOff" advanced="false">
+        <item-type>Switch</item-type>
+        <label>Switch off fan at night</label>
+    </channel-type>
+    
+    <channel-type id="nightmode_WifiNightOff" advanced="false">
+        <item-type>Switch</item-type>
+        <label>Switch off WLAN at night</label>
+    </channel-type>
+     
+    <channel-type id="devicename" advanced="false">
+        <item-type>String</item-type>
+        <label>Device Name</label>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="RoomType" advanced="false">
+        <item-type>String</item-type>
+        <label>Type of room</label>
+    </channel-type>
+    
+    <channel-type id="Logging" advanced="false">
+        <item-type>String</item-type>
+        <label>Logging level</label>
+    </channel-type>
+    
+    <channel-type id="DeleteKey" advanced="false">
+        <item-type>String</item-type>
+        <label>Settings to be deleted</label>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="FireAlarm" advanced="false">
+        <item-type>Switch</item-type>
+        <label>Send Fire Alarm if certain levels are met</label>
+    </channel-type>
+    
+    <channel-type id="air-Q-Hardware-Version" advanced="false">
+        <item-type>String</item-type>
+        <label>Hardware version</label>
+        <state readOnly="true"></state>
+    </channel-type>
+      
+    <channel-type id="WLAN_config_Gateway" advanced="false">
+        <item-type>String</item-type>
+        <label>Network Gateway</label>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="WLAN_config_MAC" advanced="false">
+        <item-type>String</item-type>
+        <label>MAC Address</label>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="WLAN_config_SSID" advanced="false">
+        <item-type>String</item-type>
+        <label>WLAN SSID</label>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="WLAN_config_IPAddress" advanced="false">
+        <item-type>String</item-type>
+        <label>Assigned IP address</label>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="WLAN_config_NetMask" advanced="false">
+        <item-type>String</item-type>
+        <label>Network mask</label>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="WLAN_config_BSSID" advanced="false">
+        <item-type>String</item-type>
+        <label>Network BSSID</label>
+        <state readOnly="true"></state>
```

Comment:
Excellent input, now I am checking which channels are better suited as meta data (properties), however I encounter one problem: the length of the property string is limited. For example, the list of available sensors would be well suited for a property because it obviously describes the properties of the air-Q device, but the corresponding string is limited in: `o3, no2, so2, co, tvoc, oxygen, particulates, sound, co2, temperature, pressure, humidity, humidity_abs, dewpt` (the string would include several more)
Any idea how I can solve this? Inserting new lines (`\n` or `<br />`) does not work.